conu/backend/k8s/service.py

```diff
@@ -104,8 +104,12 @@ def get_ip(self):
         return self.spec.cluster_ip
 
     def create_in_cluster(self):
+        """
+        call Kubernetes API and create Service in cluster, raise ConuExeption if API call fail
```

Comment:
*call Kubernetes API and create this Service in cluster, raise ConuExeption if the API call fails*